Gruntfile.js

```diff
@@ -233,8 +222,13 @@ module.exports = function(grunt) {
         'sentinel/**/*.js',
         'shared-libs/**/*.js',
         'admin/**/*.js',
-        'ddocs/**/*.js',
-      ],
+        '!ddocs/**/*.js',
```

Comment:
So we couldn't work out how to configure ESLint for ddocs?

Can we put a comment here pointing out that we *do* want to lint ddocs but we haven't worked out how to get ESLint to lint them properly.

Reply:
Sadly we couldn't, I'll add a comment and if possible we can open a related ticket to this issue specifically